doc/source/conf.py

```diff
@@ -40,15 +39,63 @@ def preprocess_notebooks(app: Sphinx, *args, **kwargs):
 
     print("Converting Markdown files to IPyNB...")
     for path in (HERE / "regression").glob("*.md"):
+        if path.match("regression/header.md"):
+            continue
         nb = jupytext.read(str(path))
+
+        # In .md to .ipynd conversion, do not include any cells that have the
+        # jupyterlite_sphinx_strip tag
+        nb.cells = [
+            cell for cell in nb.cells if "jupyterlite_sphinx_strip" not in cell.metadata.get("tags", [])
+        ]
+
```

Comment:
Here, we can drop the entire functionality and use `jupyterlite-sphinx` to do this natively; should I add those changes here for you to rebase your branch over them, or do you wish to do so yourself?

Reply:
Hmm, at what point does the JupyterLite Sphinx extension strip the tagged cells? Does it happen at render time in the Lite app or does it happen when converting .md to .ipynb?

I think that the handcoded outputs should be stripped in either case, whether loading the notebook in Lite or downloading it as a .ipynb file. 

It is perhaps a mistake, though, to use the `jupyterlite_sphinx_strip` tag here to remove the cell during markdown to ipynb conversion. Perhaps I should do a repo-wide search "jupyterlite_sphinx_strip" and replace it with something like "pywt-remove-from-ipynb"... what do you think?